Emby.Server.Implementations/Syncplay/SyncplayController.cs

```diff
@@ -505,7 +529,8 @@ private void HandleBufferingDoneRequest(SessionInfo session, PlaybackRequest req
         private void HandlePingUpdateRequest(SessionInfo session, PlaybackRequest request)
         {
             // Collected pings are used to account for network latency when unpausing playback
-            _group.UpdatePing(session, request.Ping ??= _group.DefaulPing);
+            var ping = request.Ping ?? _group.DefaulPing;
+            _group.UpdatePing(session, ping);
```

Comment:
why did you want to do this? or is it Github showing something strange?

Reply:
You mean the extra `var`? I thought it had something to do with casting but the compiler is not complaining.
I can't really recall :thinking:. I'll make it a single line.